ironfish/src/account/account.ts

```diff
@@ -16,6 +16,7 @@ export class Account {
   readonly outgoingViewKey: string
   publicAddress: string
   rescan: number | null
+  upToDate: boolean
```

Comment:
I feel weird about this value living in an account. An account on its own wouldn't know if its up to date. What do you think about having a check in `accounts.ts`, something like ` function isUpToDate(account: Account): boolean`?